app/views/dashboard/index.njk

```diff
@@ -15,11 +16,20 @@ Dashboard - {{currentService.name}} {{currentGatewayAccount.full_type}} - GOV.UK
         If you use the GOV.UK Pay API, you will need to <a href="{{ apiKeysLink }}">create new API keys</a>.
       </p>
     {% endset %}
-    {{ govukNotificationBanner({
-      html: html,
-      type: 'success'
-    }) }}
+    <div class="govuk-grid-column-full">
+      {{ govukNotificationBanner({
+        html: html,
+        type: 'success'
+      }) }}
+    </div>
   {% endif %}
+
+  <div class="govuk-grid-column-full">
+    {% if messages is defined and messages is iterable and messages|length > 0 %}
+      {{ systemMessages({ messages: messages }) }}
+    {% endif %}
+  </div>
+
```

Comment:
Is there any way both of these banners can show at the same time? May be a bit out of scope but ideally the system messages banner could encompass both use cases

Reply:
possibly, however the non-systemMessages banner is only temporary afaik (shown when s stripe test account is requested from the dashboard) so am not inclined to put in the effort to change it if it's being removed